filebeat/input/unix/input.go

```diff
@@ -71,17 +93,27 @@ func (s *server) Test(_ input.TestContext) error {
 }
 
 func (s *server) Run(ctx input.Context, publisher stateless.Publisher) error {
-	log := ctx.Logger.Named("input.unix").With("path", s.config.Config.Path)
+	log := ctx.Logger.With("path", s.config.Config.Path)
```

Comment:
[Question]

Why removing the name of the logger? The name helps with filtering, specially when enabling log debug

Reply:
Please see https://github.com/elastic/beats/pull/33930#discussion_r1041709954